packages/docs/src/routes/docs/concepts/resumable/index.mdx

```diff
@@ -15,8 +15,10 @@ When an SSR/SSG application boots up on a client, it requires that the framework
 1. Listeners - locate event listeners and install them on the DOM nodes to make the application interactive.
 2. Component tree - build up an internal data structure representing the application component tree.
 3. Application state - restore the application state.
+[//]: # (What state are we talking about ? Is there an application-wise state in Qwik ?)
```

Comment:
this means that the app ran on the server and built up state - like list of products from the server etc... if it was saved in a `store` or a `signal` on the server it is the "app state".. so it gets restored on the client